HW_0210.ipynb

```diff
@@ -295,6 +295,7 @@
         "def power(cars):\n",
         "    cars['has_power'] = cars['Variant'].str.contains(\"Power\")\n",
         "    return cars\n",
+        "# REVIEW COMMENT: The goal here was to write a regular function and use it in the .apply. so sth like def my_function(arg):... -> .apply(my_function)\n",
```

Comment:
@novik2713 good effort to use github. i'm adding this pr to review the homework solution. you can see the comments tagged with "REVIEW COMMENT"